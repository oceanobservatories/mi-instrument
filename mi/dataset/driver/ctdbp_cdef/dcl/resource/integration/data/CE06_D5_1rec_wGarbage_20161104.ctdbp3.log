2016/11/04 00:28:05.037 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 01:28:04.111 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 02:28:04.730 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 03:28:04.859 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 04:28:03.589 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 05:28:05.165 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 06:28:06.356 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 07:28:04.183 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 08:28:02.960 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 09:28:05.002 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 10:28:03.066 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 10:30:15.359 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/11/04 10:30:38.861 [ctdbp3:DLOGP6]:Š¢bŠ¢’ªbŠ²ŠbÊšb‚¢½ÙÍŠ‚Òš‚Ò’¢ÿ
2016/11/04 10:31:05.267 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/11/04 11:28:03.864 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 11:30:16.663 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/11/04 11:30:31.367 065, 1397, 153, 91, 04 Nov 2016 11:30:25
2016/11/04 11:31:06.562 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/11/04 12:28:04.287 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 12:30:07.779 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/11/04 12:31:00.577 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/11/04 13:28:03.163 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 14:28:07.136 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 15:28:05.084 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 16:28:04.810 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 16:30:07.396 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/11/04 16:31:08.991 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/11/04 17:28:05.139 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 18:28:04.728 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 18:30:16.452 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/11/04 18:30:39.953 [ctdbp3:DLOGP6]:Š‚‚ªbŠ²ŠbÂ¢b‚¢½ÙÚÂÒš‚Ò’ªÿ
2016/11/04 18:31:06.350 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/11/04 19:28:06.032 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 20:28:03.574 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 21:28:04.928 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 22:28:03.143 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/11/04 22:30:05.647 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/11/04 22:31:07.231 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/11/04 23:28:05.969 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
