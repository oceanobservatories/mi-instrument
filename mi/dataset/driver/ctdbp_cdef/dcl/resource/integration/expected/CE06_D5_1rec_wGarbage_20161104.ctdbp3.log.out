
 subsite | node | sensor | bin | method | time | deployment | id | conductivity | date_time_string | dcl_controller_timestamp | driver_timestamp | ingestion_timestamp | internal_timestamp | port_timestamp | preferred_timestamp | pressure | provenance | quality_flag | temp
---------+------+--------+-----+--------+------+------------+----+--------------+------------------+--------------------------+------------------+---------------------+--------------------+----------------+---------------------+----------+------------+--------------+------

(0 rows)
